RELEASES.md

```diff
@@ -115,12 +135,6 @@ Note that branches and release from before 1.0 may not follow these rules.
 |                        | v1.2 (End-Of-Life) | 1.10+              | v1alpha2     |
 |                        | v1.3 (End-Of-Life) | 1.12+              | v1alpha2     |
 |                        | v1.4 (End-of-Life) | 1.19+              | v1alpha2     |
```

Comment:
The tables should be consistently sorted in ascending or descending order

Reply:
Updated, I think we should change the ordering but agree on consistency